### PR TITLE
Untested: Added --dry as an option.

### PR DESCRIPTION
**Pull request by jacob9706**
This will print the command that will be ran and prompt for yes or no if a command is found.
Example:
`fuck --dry`

### CODE REVIEW

**Comment by jamescostian** [comment]
:+1: for this solution that prevents heart attacks (#1)

I don't always fuck, but when I do, I fuck --dry

**Comment by ahmetalpbalkan** [comment]
Dry fuck doesn't sound like a good idea at all. Kills the point of the project.

How about printing the command that's going to be executed, sleeping 1.5 or 2 seconds to give a chance to ctrl+c? `git` autocorrect config does this way and works for most. https://coderwall.com/p/gvw46q/git-autocorrect-spelling

**Comment by hmemcpy** [comment]
Is this like dry humping then?

File: thefuck/main.py [hunk 1]
```diff
@@ -90,6 +95,12 @@ def main():
         rules = get_rules(user_dir, settings)
         matched_rule = get_matched_rule(command, rules, settings)
         if matched_rule:
+            print(rule.get_new_command(command, settings))
+            
+            if opts.do_it_dry:
+                answer = input('Run command? [Y/n]: ')
```
Comment (kilotaras) [inline thefuck/main.py]: Can empty answer be treated as yes here?

**Comment by foofoo112** [comment]
Yyy

**Comment by SkyRocknRoll** [comment]
I Would like to Have this 'Run command? [Y/n] 

+1

**Comment by nwinkler** [comment]
If a question is asked, it doesn't sound like _dry_ mode to me. With other tools, _dry_ mode doesn't do anything.

The above ("Run command? [Y/n]") looks more like interactive mode to me, so a `-i` instead of `-dry` might be more appropriate.

**Comment by SkyRocknRoll** [comment]
if i want to use this for serious command which modifies or deletes my system data then it would be nice to have confirmation.

**Comment by nvbn** [comment]
I added settings option `require_confirmation`, I think it's what you want, when It's `True`:
```bash
➜  apt-get install vim    
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?
➜  fuck
sudo apt-get install vim [Enter/Ctrl+C]
[sudo] password for nvbn: 
```
Or:
```bash
➜  fuck
sudo apt-get install vim [Enter/Ctrl+C]^CAborted
```

**Pull request closed without merging**